src/Raven.Server/Documents/ETL/Providers/Elasticsearch/ElasticSearchLoadException.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Raven.Server.Documents.ETL.Providers.ElasticSearch
```

Comment:
move it to Raven.Server.Exceptions.ETL.ElasitcSearch ?

Reply:
done